workflow/run_simulation.rb

```diff
@@ -64,6 +65,17 @@ def run_workflow(basedir, rundir, hpxml, debug, timeseries_output_freq, timeseri
   args['output_format'] = output_format
   update_args_hash(measures, measure_subdir, args)
 
+  # Add utility bills measure to workflow
+  if add_utility_bills
+    measure_subdir = 'ReportUtilityBills'
+    args = {}
+    args['output_format'] = output_format
+    args['wood_cord_marginal_rate'] = 0.015
+    args['wood_pellets_marginal_rate'] = 0.015
+    args['coal_marginal_rate'] = 0.015
```

Comment:
Why are these here? Can't they be default argument values in ReportUtilityBills/measure.rb, as is done for every other argument?

Reply:
I guess it just didn't click for me that your previous comment was referring to _measure_ arguments.